plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClient.java

```diff
@@ -191,8 +192,8 @@
             .buildOrThrow();
 
     private final boolean synonymsEnabled;
-    private final ConnectorExpressionRewriter<String> connectorExpressionRewriter;
-    private final AggregateFunctionRewriter<JdbcExpression, String> aggregateFunctionRewriter;
+    private final ConnectorExpressionRewriter<ParameterizedExpression> connectorExpressionRewriter;
+    private final AggregateFunctionRewriter<JdbcExpression, ?> aggregateFunctionRewriter;
```

Comment:
why not `AggregateFunctionRewriter<JdbcExpression, ParameterizedExpression>`?

Reply:
because the second parameter matters only for `AggregateFunctionRewriter` construction, but not its usage